Tasks/Common/ServiceFabricHelpers/CommonHelpers.ps1

```diff
@@ -0,0 +1,64 @@
+function Publish-Telemetry
+{
+    Param (
+        [String]
+        $TaskName,
+
+        [System.Management.Automation.ErrorRecord]
+        $ErrorData
+    )
+
+    try
+    {
+        $telemeteryData = @{
+            'OperationId'   = $global:operationId
```

Comment:
Should this function explicitly take operationId as input instead global.operationId

Reply:
Can do.